gatsby-config.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
       resolve: `gatsby-source-filesystem`,
       options: {
         path: `./src/data/`,
+        google: {
+          families: ['Roboto']
+        }
       },
     },
     `gatsby-transformer-sharp`,
```

Comment:
It seems this is not the right way to do so, after including `gatsby-plugin-web-font-loader` Plugin. You can read the [docs](https://www.gatsbyjs.com/plugins/gatsby-plugin-web-font-loader/).

```suggestion
    `gatsby-transformer-sharp`,
    {
      resolve: 'gatsby-plugin-web-font-loader',
      options: {
        google: {
          families: ['Droid Sans', 'Droid Serif']
        }
      }
    }
```
> this is how we can achieve the required results.

Reply:
But that are just different fonts from google, I tested those three "Roboto", "Droid Sans" and "Droid Serif" and Roboto was the best, but I'll also add the other two

Reply:
![Screen Capture_select-area_20201003164514](https://user-images.githubusercontent.com/19675651/95000529-fbf0e080-0597-11eb-98b9-bab3f4a03be8.png)

Reply:
I was not talking about changing `font-families`. The point here is, the way you have added the `google fonts` is not the correct way as you have added a plugin for that `gatsby-plugin-web-font-loader` and as per the [docs](https://www.gatsbyjs.com/plugins/gatsby-plugin-web-font-loader/), it should be done in this way:

```
module.exports = {
  plugins: [
    {
      resolve: 'gatsby-plugin-web-font-loader',
      options: {
        google: {
          families: ['Droid Sans', 'Droid Serif']
        }
      }
    }
  ]
}
```

> Hope, this makes sense. Also, add a screenshot, that showcases how the font looks when you run the web app locally.

Reply:
I'm feeling so stupid right now. Sorry for that, I realized now what were u talking about. Anyway I made the changes